github/github.go

```diff
@@ -986,6 +996,59 @@ func (g *GitHub) UpdateReleasePage(
 	return release, nil
 }
 
+// toRepositoryRelease builds a repository release from the set of options
```

Comment:
tiny nit
```suggestion
// toRepositoryRelease builds a repository release from the set of options.
```

other comments are affected as well.

Reply:
done!